hyperframe/frame.py

```diff
@@ -381,20 +391,21 @@ def __init__(self, stream_id=0, settings=None, **kwargs):
         self.settings = settings or {}
 
     def serialize_body(self):
-        settings = [struct.pack("!HL", setting & 0xFF, value)
-                    for setting, value in self.settings.items()]
-        return b''.join(settings)
+        return b''.join([_STRUCT_HL.pack(setting & 0xFF, value)
+                         for setting, value in self.settings.items()])
```

Comment:
If you're curious how much faster, quick benchmark suggests about **51%** faster for short sequences.